lib/config/src/expression_functions.rs

```diff
@@ -6,6 +6,7 @@ use crate::select_parser::ProviderStream;
 
 use ether::{Either, Either3, EitherExt};
 use futures::{stream, Stream, StreamExt, TryStreamExt};
+use js_sys;
```

Comment:
Clippy is complaining here because on line 419 you use the fully qualified path to the `Date.now` method, so this `use` is unnecessary.